test/token-helper.test.js

```diff
@@ -10,15 +10,44 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+const mockSupports = jest.fn()
+const mockImsLogin = jest.fn()
+
+const setImsPluginMocks = (supports, token) => {
+  if (typeof supports === 'function') {
+    mockSupports.mockImplementation(supports)
+  } else {
+    mockSupports.mockReturnValue(supports)
+  }
+
+  if (typeof token === 'function') {
+    mockImsLogin.mockImplementation(token)
+  } else {
+    mockImsLogin.mockReturnValue(token)
+  }
+}
+
+jest.mock('@adobe/aio-lib-ims-jwt', () => ({
```

Comment:
out of curiosity why are you only mocking lib-ims-jwt and not the other plugins?

Reply:
Good point. All the tests used 'custom' plugins to handle the tests for the plugin interface. Since we don't have custom plugin support anymore I had to pick one of the default plugins to mock to fit the existing tests. Now that I re-think it, I should discard all the tests and re-do for the default plugins

Reply:
fixed to use default ims plugins